tensorflow-notebook/README.md

```diff
@@ -0,0 +1,131 @@
+![docker pulls](https://img.shields.io/docker/pulls/jupyter/tensorflow-notebook.svg) ![docker stars](https://img.shields.io/docker/stars/jupyter/tensorflow-notebook.svg)
+
+# Jupyter Notebook Scientific Python Stack + Tensorflow
+
+## What it Gives You
+
+* Everything in [Scipy Notebook](https://github.com/jupyter/docker-stacks/tree/master/scipy-notebook)
+* Tensorflow for Python 2.7 and 3.5 (without GPU support)
+
+
+## Basic Use
+
+The following command starts a container with the Notebook server listening for HTTP connections on port 8888 without authentication configured.
+
+```
+docker run -d -p 8888:8888 jupyter/tensorflow-notebook
+```
+
+## Tensorflow Single Machine Mode
+
+As distributed tensorflow is still immature, we currently only provide the single machine mode. Will add the distributed one soon.
```

Comment:
Leave off the second sentence. If it comes later, that's great. Let's not promise at the moment.